include/RAJA/policy/cuda/kernel/CudaKernel.hpp

```diff
@@ -58,7 +58,7 @@ namespace RAJA
  * runtime.
  * Num_threads is 1024, which may not be appropriate for all kernels.
  */
-template <bool async0, int num_blocks, int num_threads>
+template <bool async0, size_t num_blocks, int num_threads>
```

Comment:
why were num_blocks  changed to size_t?  if that makes sense, why not change num_threads to size_t?

Reply:
This had to be done since the product of num_blocks in x,y,z can overflow an int type. This won't happen with the total number of threads per block.